src/index.css

```
@@ -123,4 +123,36 @@
+    background-color: var(--background-hover);
+    border-radius: 3px;
+  }
+
+  .scrollbar-thin::-webkit-scrollbar-thumb:hover {
+    background-color: var(--text-muted);
+  }
+
+  /* Enhanced input focus styles */
+  .input-focus-ring:focus {
+    outline: none;
+    ring: 2px;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)